### PR TITLE
feat: add quickstart doc

### PR DESCRIPTION
**Pull request by Randolph314**
## Description

Closes [#215](https://mask.atlassian.net/browse/MF-215)

## Type of change

- [x] Documentation
- [ ] Code refactoring (Restructuring existing code w/o changing its observable behavior)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (a fix or feature that would make something no longer possible to do/require old user must upgrade their Mask Network to this new version)

## Checklist

- [ ] My code follows the style guidelines of this project.
- [ ] I have performed a self-review of my own code.
  - [ ] I have removed all in development `console.log`s
  - [ ] I have removed all commented code.
- [ ] I have commented on my code, particularly in hard-to-understand areas.
- [ ] I have read [Internationalization Guide](https://github.com/DimensionDev/Maskbook/blob/develop/docs/i18n-guide.md) and moved text fields to the i18n JSON file.

If this PR depends on external APIs:

- [ ] I have configured those APIs with CORS headers to let extension requests get passed. 
  - chrome extension: `chrome-extension://[id]`
  - firefox extension: `moz-extension://[id]`
- [ ] I have delegated all web requests to the background service via the internal RPC bridge.

### CODE REVIEW

File: docs/plugin-development-quickstart.md [hunk 1]
```diff
@@ -4,28 +4,185 @@ author: Randolph
...
+
+You can create a plugin folder by entering following command
 
 > npx gulp new-pkg
 
-...(a brief introduction about the files generated)
+After entering related information about the plugin you want to create, a plugin folder will be created in `packages/plugins`. This folder contains lots of files, and core files are `SNSAdaptor` and `Worker`. The former contains all the code related to front-end interaction, the latter plays a database-like role.
+Since we want to add a application into maskbook, we need to add related config:
+
+`pnpm-lock.yaml`
```
Comment (septs) [inline docs/plugin-development-quickstart.md]: ???

File: docs/plugin-development-quickstart.md [hunk 2]
```diff
@@ -4,28 +4,185 @@ author: Randolph
...
+    "@masknet/plugin-ENS": "workspace:*",
+```
+
+`packages/mask/.webpack/config.ts`
+
+```JavaScript
+    '@masknet/plugin-ENS': join(__dirname, '../../plugins/your file name/src/'),
+```
+
+`packages/mask/src/plugin-infra/register.js`
```
Comment (septs) [inline docs/plugin-development-quickstart.md:20]: `.js`?

Comment (Jack-Works) [reply docs/plugin-development-quickstart.md:20]: Yes. This is a js file for some TypeScript reason

File: docs/plugin-development-quickstart.md (hunk 1 above)
Comment (Jack-Works) [reply docs/plugin-development-quickstart.md]: Hmm, wait. Why you're suggesting editing the lock file directly? It should be changed by pnpm itself only.

File: docs/plugin-development-quickstart.md [hunk 3]
```diff
@@ -4,28 +4,107 @@ author: Randolph
...
+    import '@masknet/plugin-ENS'
+```
+
+## 3. How to inject UI
+
+We will inject our UI in three places of twitter to provide different services. And `Plugin.SNSAdaptor.Definition` in `SNSAdaptor` is where we inject our UI mainly.
+
+### 3.1 Add UI in application board
+
+Add following config in `ApplicationBoard.tsx`：
```
Comment (Jack-Works) [inline docs/plugin-development-quickstart.md]: this is outdated after https://github.com/DimensionDev/Maskbook/pull/5939 lands

**Review by Jack-Works — CHANGES_REQUESTED** [review]

File: packages/plugins/ENS/src/SNSAdaptor/ENSDialog.tsx [hunk 4]
```diff
@@ -4,7 +4,7 @@ import { Box } from '@mui/system'
 import { ChangeEvent, useCallback, useState } from 'react'
 import { TextField, Button } from '@mui/material'
 import { useAccount } from '@masknet/web3-shared-evm'
-import { hash } from 'eth-ens-namehash'
+const { hash } = require('eth-ens-namehash')
```
Comment (septs) [inline packages/plugins/ENS/src/SNSAdaptor/ENSDialog.tsx:7]: ?

File: .i18n-codegen.json [hunk 5]
```diff
@@ -182,6 +182,18 @@
                 "sourceMap": "inline"
             }
         },
+        {
+            "input": "./packages/plugins/ENS/src/locales/en-US.json",
```
Comment (Jack-Works) [inline .i18n-codegen.json:186]: why you're adding a new package in "quickstart doc"?

**Comment by Jack-Works** [comment]
please rename this PR, it is not "add quickstart doc". And fix merge conflicts plz

**Pull request closed without merging**